exercises/rna-transcription/source/rnatranscription/RNAtest.ceylon

```diff
@@ -0,0 +1,31 @@
+import ceylon.test { ... }
+
+// Tests adapted from x-common version 1.0.0
+{[String, String|Error]*} cases => {
+  // empty strand
```

Comment:
this case is not present in https://github.com/exercism/problem-specifications/blob/master/exercises/rna-transcription/canonical-data.json#L48-L51 . shall it be added to canonical-data.json?

Reply:
has now been added in https://github.com/exercism/ceylon/pull/73 because https://github.com/exercism/problem-specifications/pull/1266